pkg/kritis/review/review.go

```diff
@@ -175,42 +258,40 @@ func (r Reviewer) handleViolations(image string, pod *v1.Pod, violations []polic
 	return fmt.Errorf(errMsg)
 }
 
-// Create attestations for 'image' by all 'auths'.
-func (r Reviewer) addAttestations(image string, isp v1beta1.ImageSecurityPolicy, auths []v1beta1.AttestationAuthority, c metadata.ReadWriteClient) error {
+// Create attestations for 'image' by ISP auth.
+func (r Reviewer) addAttestation(image string, isp v1beta1.ImageSecurityPolicy, auth v1beta1.AttestationAuthority, c metadata.ReadWriteClient) error {
 	errMsgs := []string{}
-	for _, a := range auths {
-		// Get or Create Note for this this Authority
-		n, err := util.GetOrCreateAttestationNote(c, &a)
-		if err != nil {
-			errMsgs = append(errMsgs, err.Error())
-		}
-		// Get secret for this Authority
-		s, err := r.config.Secret(isp.Namespace, a.Spec.PrivateKeySecretName)
-		if err != nil {
-			errMsgs = append(errMsgs, err.Error())
-		}
-		// Create Attestation Signature
-		if _, err := c.CreateAttestationOccurrence(n, image, s, grafeas.DefaultProject); err != nil {
-			errMsgs = append(errMsgs, err.Error())
-		}
-
+	// Get or Create Note for this this Authority
+	n, err := util.GetOrCreateAttestationNote(c, &auth)
+	if err != nil {
+		errMsgs = append(errMsgs, err.Error())
```

Comment:
We want to exit early in this case, right?

Reply:
done.